sa/model.go

```diff
@@ -621,5 +633,15 @@ func modelToAuthzPB(am *authz2Model) (*corepb.Authorization, error) {
 			pb.Challenges = append(pb.Challenges, challenge)
 		}
 	}
+	// Populate combinations
```

Comment:
I thought the `Combinations` stuff was a legacy of earlier ACME v1 drafts, is that not the case?

Reply:
We still send them to the user in the v1 API unfortunately, otherwise I'd be it.

Reply:
Ahhh blah. I guess there are clients that might be relying on it :-( Ok

Reply:
Note: We could move the "populate challenges" code to the display layer for wfe1, and always populate a fixed list of challenges in order.